recipes/002_registration.py

```diff
@@ -88,11 +88,11 @@
       - `DemoSampler2`
       - `demo_function1`
 
-- An `Installation <https://github.com/optuna/optunahub-registry/blob/main/template/README.md#installation>`__ section that describes how to install the additional dependencies if required. For example:
+- An `Installation <https://github.com/optuna/optunahub-registry/blob/main/template/README.md#installation>`__ section that describes how to install the additional dependencies if required. If your package contains ``requirements.txt``, it will be available at ``https://hub.optuna.org/package/{category}/{your_package_name}/requirements.txt``. Then, the package dependencies can be installed as follows.
```

Comment:
`package` is not included in the URL.
```suggestion
- An `Installation <https://github.com/optuna/optunahub-registry/blob/main/template/README.md#installation>`__ section that describes how to install the additional dependencies if required. If your package contains ``requirements.txt``, it will be available at ``https://hub.optuna.org/{category}/{your_package_name}/requirements.txt``. Then, the package dependencies can be installed as follows.
```